### PR TITLE
chore: Update README with intro and status

### PR DESCRIPTION
**Pull request by pauldix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # InfluxDB IOx
 
-TODO: Blog post summary here
+InfluxDB IOx (short for Iron Oxide, pronounced InfluxDB "eye-ox") is the future core of InfluxDB. The name is in homage to Rust, the language this project is written in. It is built using [Apache Arrow](https://arrow.apache.org/) and [DataFusion](https://arrow.apache.org/blog/2019/02/04/datafusion-donation/) among other things. InfluxDB IOx aims to be:
+
+* The future core of InfluxDB supporting industry standard SQL, InfluxQL, and Flux
```
Comment (e-dard) [inline README.md:5]:
```suggestion
* The future core of InfluxDB; supporting industry standard SQL, InfluxQL, and Flux
```

Add semicolon.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+* An in-memory columnar store using object storage for persistence
+* An fast analytic database for raw events (like tracing data)
```
Comment (e-dard) [inline README.md:7]: `and logs`?

**Review by e-dard — APPROVED** [review]
LGTM 👍  I had a couple of tiny suggestions.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+* A system for defining replication and partitioning rules for InfluxDB time series data and tabular analytics data
+* A system supporting real-time subscriptions
+* A processor that can transform and do arbitrary computation on time series and event data as it arrives
+* An analytic database built for data science, supporting Apache Arrow Flight for fast data transfer
+
+Persistence is through Parquet files in object storage. It is a design goal to support integration with other big data systems through object storage and Parquet specifically. For more deatails on the motivation behind the project and some of our goals, read through the [InfluxDB IOx announcement blog post](https://www.influxdata.com/blog/announcing-influxdb-iox/).
```
Comment (alamb) [inline README.md:13]: We could also throw in a mention of the original Arrow / parquet / flight blog too: https://www.influxdata.com/blog/apache-arrow-parquet-flight-and-their-ecosystem-are-a-game-changer-for-olap/

**Review by alamb — APPROVED** [review]
Let's ship it!

**Pull request merged**